components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -252,6 +252,12 @@ public void deleteUser(String userId) throws NotFoundException, CharonException
             } else {
                 //we assume (since id is unique per user) only one user exists for a given id
                 userName = userNames[0];
+                //check if SCIM is enabled for the user store
```

Comment:
Move the comment to a line below, just before the if condition.

Reply:
3b71977a5ec04cda298019f05419d4b051dcd457